homeassistant/components/fitbit/strings.json

```
@@ -38,7 +38,101 @@
       },
       "battery_level": {
         "name": "Battery level"
+      },
+      "activity_calories": {
+        "name": "Activity Calories"
+      },
+      "calories": {
+        "name": "Calories"
+      },
+      "calories_bmr": {
+        "name": "Calories BMR"
+      },
+      "distance": {
+        "name": "Distance"
+      },
+      "elevation": {
+        "name": "Elevation"
+      },
+      "floors": {
+        "name": "Floors"
+      },
+      "resting_heart_rate": {
+        "name": "Resting Heart Rate"
```

Names should be sentence case
```suggestion
        "name": "Resting heart rate"
```